wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -1,117 +1,47 @@
 
-Visual content (aka: beatmap backgrounds, skins, objects and other included graphical or video-related assets distributed with a beatmapset) **MUST NOT** contain any of the following:
+## Criteria
 
-### Excessive violence
+- **Images with any sexual content involving (or are targeted at) minors are not allowed.**
+- **Images with excessive violence are not allowed.**
+- **Images with significant sexual innuendo are not allowed.**
+- **Images with explicit content or sexual posturing are not allowed.**
+- **Images with any erotic content or graphic displays of sexuality are not allowed.**
+- **Images with any examples of drug use or preparation are not allowed.**
+- **Images with depictions of any deliberately inflammatory political, cultural, religious or social content are not allowed.**
 
-This includes violence towards humans, humanoid characters or animals of any kind.
+## Exceptions
 
-Any imagery that portrays gore, mutilation, maiming, severe wounding or graphical depictions of recent death with an obvious cause present (such as hanging, choking or bleeding out) is not acceptable or to be tolerated.
+- Images with mild to moderate depictions of violence may be allowed.
+- Images with depictions of artistic nudity in the context of classical artworks or other tasteful depictions of the human body may be allowed.
+- Images with depictions of mild romance, flirting or intimate affection may be allowed.
 
-### Sexual innuendo
+## Getting assessed
 
-This refers to the mimicry of sexual acts or soliciting in a lascivious or lewd manner. Content of this nature in the anime context is often referred to as "ecchi".
+To check to see if an image is appropriate, **please submit a formal request to any Beatmap Nominator (for beatmap related content) or member of the Global Moderation Team (for anything).**
 
-Any imagery that imitates sexual acts or involves objects or mannerisms that mimic or reference any genitalia (such as phallic objects) is not allowed.
+The issue will then be raised in a dedicated content reviewing channel for discussion and a voting period.
 
-### Explicit or sexual posturing
+The voting period will not last longer than 7 days.
 
-Any imagery that is highly suggestive or directly implies imminent sexual intent or anticipation is not allowed. This includes "beckoning" or "come hither" type imagery which is inviting or otherwise highly suggestive of imminent sexual contact.
+The vote must pass with at least a 60% majority. Votes that do not gain this majority will be considered too contentious to safely allow, and will be automatically denied.
 
-This also applies to "accidental" posturing, such as imagery depicting characters being caught unaware during bathing, or other such situations.
+If a given image does not pass a vote, that decision is **final** and may not be overridden or raised for discussion again, unless the image is altered to attempt to resolve the problems with it.
```

Comment:
you've been talking about an example vote scenario, so I think it'd be better to talk about "the" situation rather than introducing other hypothetical ones.

(i can't write this as a suggestion cuz github)
```markdown
The image will be allowed if the vote passes with at least a 60% majority. Otherwise, it will be considered too contentious to safely allow, and automatically denied. The decision by vote is **final** and may not be overridden or raised for discussion again, unless the image is altered in attempt to resolve the problems with it.
```